### DISCUSSION TITLE
Review of all Governor parameters

### DISCUSSION THREAD

**Discussion by tbjump** [wormhole-foundation/wormhole#3259] [Governance]
It's been a while since governor parameters for all chains were set. This discussion is to do a systematic review and adjustment of all governor parameters. 

The accepted practice for determining Governor parameters is:
* Set the daily not higher than what are reasonably expected daily outflows. E.g. the daily should not be higher than recent monthly averages.
* A chain's security posture (e.g. track record, security assumptions, audits, bug bounty program, etc.) should influence the maximum governor limit that should be acceptable for that chain.
* The daily for any connected chains should be at least $200k, unless justified otherwise.
* The single-transfer threshold is set at 10% of the aggregate daily threshold ("daily").

I propose to increase the minimum daily from $200k to $500k. There was a proposal to increase it to $1M that received some, but not overwhelming, support. $500k should be a compromise. 

Furthermore, adjusting the individual values as follows: 

| ID | Chain       | 30d volume    | 90d volume    | TVM(*)  |  TVL           | Current daily | Proposed daily |
|----|-------------|---------------|---------------|--------|------|---------------|----------------|
| 1  | Solana      | $25M          | $86M          | 137,859,721 | 15,937,926  | 25M           | no change |
| 2  | Ethereum    | $30M          | $280M         | 15,871,976|254,570,127  | 50M           | no change |
| 3  | Terra       | $5k           | $251k         | 2,344,889|10,093,239   | 500k          | no change |
| 4  | BSC         | $56M          | $70M          | 11,498,819|9,467,466   | 5M            | no change |
| 5  | Polygon     | $3M           | $5M           | 9,279,818|4,915,057   | 5M            | reduce to 2M |
| 6  | Avalance    | $1.1M         | $5M           | 1,052,156|1,222,723    | 5M            | reduce to 1M |
| 7  | Oasis       | $176k         | $2.4M         | 1,392,294|30,541    | 2M            | reduce to 500k |
| 8  | Algorand    | $14M          | $14M          | 1,195,985|11,992    | 200k          | increase to 1M |
| 9  | Aurora      | $0            | $0            | 762|4,500        | 500k          | no change |
| 10 | Fantom      | $616k         | $1M           | 375,248|477,614      | 500k          | no change |
| 11 | Karura      | $23k          | $57k          | 110,432|4      | 500k          | no change |
| 12 | Acala       | $1.3M         | $1.6M         | 7,442,689|632    | 500k          | no change |
| 13 | Klaytn      | $20k          | $41k          | 166,693|45,769      | 5M            | reduce to 500k |
| 14 | Celo        | $1.6M         | $7M           | 21,190,473|285,305   | 5M            | reduce to 2M |
| 15 | Near        | $0            | ? (**)         | 101|6,500,177    | 5M            | reduce to 500k |
| 16 | Moonbeam    | $5M           | $11M          | 11,349,909|2,439   | 5M            | no change |
| 18 | Terra 2     | $13           | $5k           | 29,454|2,021       | 500k          | no change |
| 19 | Injective   | $97k          | $173k         | 445,319|0      | 200k          | increase to 500k |
| 21 | Sui         | $3M           | $53M          | 47,004,190|0   | 5M            | no change |
| 22 | Aptos       | $1.9M         | $6M           | 34,751,142|0   | 5M            | no change |
| 23 | Arbitrum    | $2M           | $5M           | 50,856|0       | 200k          | increase to 1M |
| 24 | Optimism    | $75k          | $487k         | 1,366|270        | 200k          | increase to 500k |
| 28 | Xpla        | $0            | $0            | 0|1            | 200k          | increase to 500k |
| 30 | Base (***)        | ?            | ?            | ?|?            | 200k          | increase to 500k |
| 32 | Sei (***)       | ?            | ?            | ?|?            | 200k          | increase to 500k |
| 3104 | WH Gateway (***)       | ?            | ?            | ?|?            | 200k          | increase to 500k |

`(*)` TVM refers to "total value minted", i.e. the aggregate value of wrapped tokens that originated elsewhere on that chain. This is in contrast to TVL, i.e. the aggregate value of tokens locked on that chain. 
`(**)` The parsing of NEAR transfers seems to not be working on Wormholescan, therefore we don't have this data, but we know that there were zero transfers in the last 30 days.
`(***)` Base, Sei, and WH Gateway were launched after the start of the review and will be increased to 500k, reflecting the new minimum, but without reviewing actual volumes. 

Data source: wormholescan.io, Global Accountant
Limitation: TVM/TVL calculations are based on coingecko market price for a list of allow-listed tokens. Without the allow-list, there would be illiquid and scam token included in the calculation, but with the allow-list it is possible that relevant tokens are missed. 
Date retrieved on 2023-08-02

If this proposal is accepted, here is the corresponding PR: https://github.com/wormhole-foundation/wormhole/pull/3260

**Comment by Micheldgit** [comment]
As Algorand 30d Volume has been $14m (placing it as the 4th with most volume) there should be a proposed increase in daily limit to at least $1m.

**Comment by claudijd** [comment]
On the $200K floor, I wonder if that low of a floor impedes some chains from really taking off because Dapp developers could get spooked by it.  What do folks think about raising the floor to say $1M?  The primary goal is to effectively set a ceiling of risk to prevent existential blow outs, in that lense, I wonder if raising that floor a little bit offers more ecosystems to grow more organically and also means that the limits require less changing until the eco system really starts to show real volume.

**Reply by pipaman** [reply]
Agree with this proposal because $20k seems very small as a limit for an individual transaction for an exchange for example. It harms the UX.

**Comment by mcf-rocks** [comment]
algorand limit looks low compared to it's volume, is there reason for that... nothing else stands out afaics

**Comment by tbjump** [comment]
Thanks all for the feedback. I updated the proposal as a compromise between the different opinions and adjusted the PR https://github.com/wormhole-foundation/wormhole/pull/3260 accordingly.
